settings.gradle

```diff
@@ -26,20 +28,32 @@ dependencyResolutionManagement {
             library('opentelemetry-proto', 'io.opentelemetry.proto', 'opentelemetry-proto').versionRef('opentelemetry')
             version('opensearch', '1.3.8')
             library('opensearch-rhlc', 'org.opensearch.client', 'opensearch-rest-high-level-client').versionRef('opensearch')
+            version('spring', '5.3.25')
+            library('spring-core', 'org.springframework', 'spring-core').versionRef('spring')
+            library('spring-context', 'org.springframework', 'spring-context').versionRef('spring')
             version('guava', '31.1-jre')
             library('guava-core', 'com.google.guava', 'guava').versionRef('guava')
         }
         testLibs {
             version('junit', '5.8.2')
             version('mockito', '3.11.2')
+            version('hamcrest', '2.2')
+            version('awaitility', '4.2.0')
+            version('spring', '5.3.25')
```

Comment:
Is there a specific reason why we need to have separate spring library for test?

Reply:
As far as I could tell, there is not a clear way to get the version from another catalog. If we find a way, I'm happy to make the change. But, this is still a great improvement in my opinion because it has the versions in only one file.